integration/member_test.go

```diff
@@ -84,7 +84,7 @@ func TestLaunchDuplicateMemberShouldFail(t *testing.T) {
 
 func TestSnapshotAndRestartMember(t *testing.T) {
 	defer testutil.AfterTest(t)
-	m := mustNewMember(t, "snapAndRestartTest", nil, nil)
+	m := mustNewMember(t, "snapAndRestartTest", nil, nil, 0)
```

Comment:
I missed this in the last PR but `quotaBackendBytes` seems out of place as an argument to this function. Changing `mustNewMember` to `mustNewMember(t *t.Testing, cfg memberConfig)` would make this `mustNewMember(t, memberConfig{name : "snapAndRestartTest})` which seems clearer to me since the quota is almost never used.